src/components/Renderer/index.tsx

```diff
@@ -155,21 +185,11 @@ const RendererBase = ({
 	});
 	const processedValue = getValue(value, schema, processedUiSchema);
 
-	if (processedValue === undefined || processedValue === null) {
-		return null;
-	}
```

Comment:
This seems to change the behavior.
Was the current behavior a bug?

Reply:
no you right, changed to value == null